Detectors/TPC/reconstruction/macro/runCATracking.C

```diff
@@ -0,0 +1,81 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <vector>
+#include <fstream>
+#include <iostream>
+#include "TSystem.h"
+
+#include "TROOT.h"
+#include "TFile.h"
+#include "TTree.h"
+#include "TChain.h"
+#include "TClonesArray.h"
+
+#include "TPCBase/Defs.h"
+#include "TPCBase/CRU.h"
+#include "TPCBase/Sector.h"
+#include "TPCBase/Mapper.h"
+#include "TPCBase/PadRegionInfo.h"
+#include "TPCSimulation/Cluster.h"
+#include "TPCReconstruction/TPCCATracking.h"
+#include "TPCReconstruction/SyncPatternMonitor.h"
+#include "TPCReconstruction/TrackTPC.h"
+#include "DetectorsBase/Track.h"
```

Comment:
since this is a macro it probably needs the usual protection `#ifdef !(__CLING__) ...` etc around the includes.

Reply:
For my culture, is it still the case with ROOT 6?